src/settings/settings.test.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @jest-environment jsdom
+ */
+const button = document.createElement("button");
+button.classList.add("section__button--save");
+document.body.append(button);
+
+const input = document.createElement("input");
+input.classList.add("section__input--playerName");
+document.body.append(input);
+
+const { setPlayerName, onButtonClick } = require("./settings");
+
+describe("Using function to set and get data from local storage", () => {
```

Comment:
`Using function`
- another describe `onButtonClick`,
- it should set player's name on button click,
- another describe `setPlayerName`,
- it should get player's name into input from storage